allauth_2fa/forms.py

```diff
@@ -28,19 +45,19 @@ def clean(self):
         return self.cleaned_data
 
 
-class TOTPDeviceForm(forms.Form):
-    token = forms.CharField(
+class TOTPDeviceForm(_TokenToOTPTokenMixin, forms.Form):
```

Comment:
I don't think this form needs this mixin since it truly is about `TOTP` tokens? You can't submit static tokens to?

Reply:
(I.e. you don't verify static tokens, you just download them for use in the future.)

Reply:
For consistency, we will want all forms' `token` fields to be henceforth named `otp_token`; the mixin warns a developer that they haven't updated their subclasses or templates correctly.